### PR TITLE
Demonstrate how to review the request body for troubleshooting

### PR DESCRIPTION
**Pull request by mptap**

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=42) <br/>All committers have signed the CLA.

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+<a name="request-body"></a>
+## Viewing the Request Body
+
+When debugging or testing, it may be useful to examine the raw request body to compare against the [documented format](https://sendgrid.com/docs/API_Reference/api_v3.html).
+
+You can do this right before you call `request.setBody(mail.build())` like so:
+
+```java
+System.out.println(mail.build());
```
Comment (mbernier) [inline TROUBLESHOOTING.md]: Would this be more direct for outputting the request body?
```
 System.out.println(request.getBody());
```

[I found this method here](https://github.com/sendgrid/java-http-client/blob/master/src/main/java/com/sendgrid/Request.java#L87)

**Review by mbernier — CHANGES_REQUESTED** [review]

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (mptap) [reply TROUBLESHOOTING.md]: @mbernier: Makes sense, updated.

**Review by mbernier — APPROVED** [review]

**Comment by thinkingserious** [comment]
Hello @mptap,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**